src/main/java/net/greatstart/validators/InvestmentValidationService.java

```diff
@@ -0,0 +1,34 @@
+package net.greatstart.validators;
+
+import net.greatstart.model.Investment;
+import net.greatstart.model.Project;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+
+@Component
+public class InvestmentValidationService {
+    private static final String WRONG_MIN_VALUE = "You entered wrong investment value. "
+            + "Min possible investment is: $";
+    private static final String WRONG_MAX_VALUE = "You entered wrong investment value. "
+            + "Max possible investment is: $";
+
+    private InvestmentValidationService() {
+    }
+
+    public static String validate(BigDecimal sum, Project project) {
```

Comment:
I copied structure of this validation service from UserValidationService and unfortunately cannot answer your question with details. ((